lib/app/routes/tracks.rb

```diff
@@ -30,6 +36,16 @@ class Tracks < Core
         problem.validate or halt 404, { error: problem.error }.to_json
         pg :problem_test, locals: { problem: problem }
       end
+
+      private
+
+      def find_track(id)
+        track = config.find(id)
+        if track.is_a?(NullTrack)
```

Comment:
Every time I see this I keep meaning to fix it. Having a NullObject and then checking type just seems silly. Not a problem that you introduced, so nothing to fix (I'm just thinking out loud).